pages/_app.tsx

```diff
@@ -1,8 +1,14 @@
+import React, { FC } from "react";
 import { HeaderNav, Footer, Helmet } from "@components";
 import { useRouter } from "next/router";
 import "../styles/main.scss";
 
-function MyApp({ Component, pageProps }) {
+interface MyAppProps {
+  Component: FC;
+  pageProps: any;
+}
+
+const MyApp: FC<MyAppProps> = ({ Component, pageProps }) => {
```

Comment:
In NextJS with Typescript, you can do this

```suggestion
import type { AppProps } from "next/app"

const MyApp: FC<MyAppProps> = ({ Component, pageProps }: AppProps) => {
```